src/net.cpp

```diff
@@ -1782,9 +1782,14 @@ void CConnman::ThreadOpenConnections(const std::vector<std::string> connect)
     // Connect to specific addresses
     if (!connect.empty())
     {
+        {
+            LOCK(m_addr_fetches_mutex);
+            if (!m_addr_fetches.empty()) {
+                LogPrintf("-seednode is ignored in -connect mode\n");
```

Comment:
Could this warning get printed if someone manually sets `-dnsseed`, since it looks like both  `-seednode` and `-dnsseed` could result in nonempty `m_addr_fetches`?

If so, would it make sense to do one or more of the following:

- Add a warning about `-dnsseed` as well
- Move the warnings to where the args are parsed

Reply:
Thanks for the review and great catch, @robot-dreams! I've moved the warning to init.cpp. It's much cleaner.